noxfile.py

```diff
@@ -156,7 +156,7 @@ def docs(session):
     shutil.rmtree(os.path.join("docs", "_build"), ignore_errors=True)
     session.run(
         "sphinx-build",
-        "-W",  # warnings as errors
```

Comment:
I remove this because there were warnings in the docs build. The session will only fail on sphinx errors.